src/private.h

```diff
@@ -48,7 +54,7 @@ typedef struct _gpuarray_blas_ops gpuarray_blas_ops;
 struct _gpucontext {
   GPUCONTEXT_HEAD;
   void *ctx_ptr;
-  void *private[7];
+  void *private[8];
```

Comment:
An enum is an int so 4 bytes.  A pointer is 4 or 8 bytes (depending on 32 or 64 bits systems).  However int can be packed together if placed one after the other so that they take a single 8-bytes slot when possible.